styleguides/stylesheets/core.md

```diff
@@ -0,0 +1,355 @@
+# Stylesheet Standards
+
+This document outlines a series of standards for creating stylesheets using:
+
+- CSS
+- SCSS
+- LESS
+
+## General Standards
+
+##### Everything should be indented using 4 spaces
+
+Bad:
+
+```css
+button {
+  background-color: red;
+  text-transform: capitalize;
+}
+```
+
+Good:
+
+```css
+button {
+    background-color: red;
+    text-transform:   capitalize;
+}
+```
+
+<small><i>Note: A tab with the size of 4 spaces will also work.</i></small>
+
+
+##### Property values should be aligned by value
+
+Bad:
+
+```css
+button {
+   background-color: red;
+   text-transform:   capitalize;
+}
+```
+
+Good:
+
+```css
+button {
+    background-color: red;
+    text-transform:   capitalize;
+}
+```
+
+##### Classes and IDs should be [kebab-case][kebab-case]
+
+Bad:
+
+```css
+.button.button_danger {
+    background-color: red;
+    text-transform:   capitalize;
+}
+```
+
+Good:
+
+```css
+.button.button-danger {
+    background-color: red;
+    text-transform:   capitalize;
+}
+```
+
+##### Multi-value properties should go below the property
+
+Bad:
+
+```css
+.button.button_danger {
+    background-color: red;
+    text-transform:   capitalize;
+    transition:       background-color ease-in-out 150ms, color ease-in-out 150ms;
+}
+```
+
+Good:
+
+```css
+.button.button-danger {
+    background-color: red;
+    text-transform:   capitalize;
+    transition:
+        background-color ease-in-out 150ms,
+        color ease-in-out 150ms;
+}
+```
+
+##### Child classes should be prefixed
+
+Bad:
+
+```css
+.button.danger {
+    background-color: red;
+    text-transform:   capitalize;
+}
+```
+
+Good:
+
+```css
+.button.button-danger {
+    background-color: red;
+    text-transform:   capitalize;
+}
+```
+
+##### Colors should be readable
```

Comment:
What is considered readable is not clear. Perhaps reword this to be more obvious?

Reply:
Ah! It seems like the good example got reset. Let me re-commit this.

Reply:
I define readable by having full length hexcodes, and spacing between `rgb`/`rgba` color parameters.

Reply:
Makes sense. It might be worth bullet pointing these.